docs/using_zeronet/create_new_site.md

```diff
@@ -25,33 +26,33 @@ $ zeronet.py
 ...
 ```
 
-- This will create the initial files for your site inside ```data/13DNDkMUExRf9Xa9ogwPKqp7zyHFEqbhC2```.
+This will create the initial files for your site inside ```data/13DNDkMUExRf9Xa9ogwPKqp7zyHFEqbhC2```.
 
 > __Note:__
 > Windows users using the bundle version must browse into the ZeroBundle/ZeroNet folder and run `"../Python/python.exe" zeronet.py siteCreate`
 
 ### 2. Build/Modify site
 
 * Update the site files located in ```data/[your site address key]``` (eg: 13DNDkMUExRf9Xa9ogwPKqp7zyHFEqbhC2).
-* When your site is ready run:
+* When finished sign your site's content:
```

Comment:
I think this should be "when finished sign*ing* your site's content:"

Reply:
Oh yes indeed, thanks for pointing that out!

Reply:
Oh, but this is in a different file. Oh well, can't hurt to put it in this PR I suppose.